### DISCUSSION TITLE
Notifying users of ppooll updates

### DISCUSSION THREAD

**Discussion by gregdl** [ppooll-dev/ppooll#115] [Ideas]
Is there a way to have users be automatically notified of any ppooll updates when they run ho_st etc? Even if it’s a small visual indicator on ho_st act with a more verbose explanation on the ppooll info maxhelp could be helpful if it’s doable.
Otherwise it’s possible (likely?) that users won’t be aware of any minor or major updates to ppooll here on github

**Comment by jpsteccato** [comment]
Great suggestion-- I've actually had some patching done for this that's been sitting on my computer.

For ppooll v8.6.0, we are implementing "automatic releases" based on the changes to the main branch.   

When an update is available, users will see the following popup:
<img width="447" alt="image" src="https://github.com/ppooll-dev/ppooll/assets/37079335/fe1adadb-0279-47d4-b0ad-8821060bb07c">
(note the new versioning system -- SEM_VERSION.RELEASE_DATE.CHANGE_NUM, where "CHANGE_NUM" is the number of releases on the given day)

What's left to do for this is implement notifications in both Discord and the ppooll mailing list, which I will work on today.

**Comment by gregdl** [comment]
that's awesome!
